src/test/java/com/epam/tc/hw9/PositiveApiTestsRestTrelloService.java

```diff
@@ -0,0 +1,97 @@
+package com.epam.tc.hw9;
+
+import io.restassured.response.Response;
+import org.assertj.core.api.Assertions;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import static org.apache.http.HttpStatus.SC_OK;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.hamcrest.Matchers.*;
+
+public class PositiveApiTestsRestTrelloService extends BaseTest {
+    @DataProvider
+    public Object[][] boardNameData() {
+        return new Object[][]{
+                {"FirstBoard"},
+                {"SecondBoard"},
+                {"ThirdBoard"},
+        };
+    }
+
+    @Test(description = "A test that verifies the correctness of creating a board",
+            dataProvider = "boardNameData")
+    public void createBoardTest(String nameBoard) {
+        restTrelloService.createNewBoard(nameBoard)
+                .then()
+                .statusCode(SC_OK);
+        boards = restTrelloService.getAllBoards();
+        Assertions.assertThat(boards.length).isEqualTo(defaultCountBoard).as("Count boards not equal expected");
+        ++defaultCountBoard;
+        assertThat(boards[0].getName()).as("The first board name does not match the test data")
+                .isEqualTo(boardNameData()[0][0]);
+    }
+
+    @Test(description = "A test that verifies the correctness of deleting the board")
+    public void deleteBoardTest() {
+        tearDown();
+        restTrelloService.createNewBoard("Board for Delete")
+                .then()
+                .statusCode(SC_OK)
+                .body("name", is("Board for Delete"));
+        boards = restTrelloService.getAllBoards();
+        assertThat(boards.length).isEqualTo(1).as("Board not create");
+        restTrelloService.deleteBoardForId(boards[0].getId())
+                .then()
+                .statusCode(SC_OK);
+        boards = restTrelloService.getAllBoards();
+        assertThat(boards.length).isEqualTo(0).as("Board not delete");
+    }
+
+    @Test(description = "A test that verifies the correctness of creating a list")
+    public void createListTest() {
+        restTrelloService.createNewBoard("Amazing board");
+        boards = restTrelloService.getAllBoards();
+        String idBoard = boards[0].getId();
+        Response resp = restTrelloService.createNewList("Amazing List", idBoard)
+                .then().statusCode(SC_OK)
+                .body("name", is("Amazing List"))
```

Comment:
do not use way of assertion swith json parser. Use DTO for data manipulation instead